src/strapvizpy/display.py

```diff
@@ -21,17 +20,17 @@ def plot_ci(sample, rep, bin_size=30, n="auto", ci_level=0.95,
     bin_size = int
         a number of bins representing intervals of equal size
         over the range
+    estimator: {"mean", "median", "var", "sd"}
```

Comment:
Add in that the default is the mean 
```suggestion
    estimator: {"mean", "median", "var", "sd"}, default = "mean"
```